Add auto update function

shoehead0125 opened the pull request.

shoehead0125 committed:
create pull reqest

shoehead0125 committed:
Make jbuilder

shoehead0125 committed:
自動更新の設定

ryouzi commented on app/assets/javascripts/message.js:
```diff
@@ -6,6 +6,29 @@ $(function(){
+    });
+  }
+  update();
+  setInterval("update()", 20000);
```
こちらの20000ですが秒数は何秒になりますか？

shoehead0125 replied:
20秒になります。

shoehead0125 committed:
set intervalの書き方の変更

さっきの書き方だとうまく関数が読み込めていなかったことに気づいたので、今回の方法に書き直しました。あと、インターバルは20秒から3秒にしています
。よろしくお願いします。

ryouzi commented on app/assets/javascripts/message.js:
```diff
@@ -6,6 +6,27 @@ $(function(){
+    .fail(function(data){
+      alert('error');
+    });
+  }, 3000);
```
3秒に1回いいですね。
チャットはリアルタイム性が大事なので、瞬時に通知が行くといいですよね。

shoehead0125 replied:
ありがとうございます。次に進みます！

ryouzi commented:
LGTM

shoehead0125 committed:
jbuilderの構造の変更

shoehead0125 committed:
特定のページのみ更新をかけるよう変更

sato11 commented on app/assets/javascripts/message.js:
```diff
@@ -6,25 +6,28 @@ $(function(){
-    });
-  }, 3000);
+  $(window).bind("load", function(){
+    if(document.URL.match(/messages/)){
```
いいアイデアですね 😊

shoehead0125 replied:
ありがとうございます。
マージはせずに、しばらく置いていました。動作に関しても問題ないので先に進みます。

sato11 reviewed (commented):
こちら一度マージしたブランチをリストアしたのでしょうか？
追加実装でしたら新しいブランチを切って作業してくださって大丈夫です！

sato11 commented:
動作が問題なければLGTMです！ 🎉
